### PR TITLE
chore(ci): Add PR template & ci actions

### PR DESCRIPTION
**Pull request by gjou-tlnd**

### CODE REVIEW

File: .github/workflows/autopublish.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+name: Faceted Search Auto Publish
+
+on:
+  push:
+    branches: [master, develop, main]
```
Comment (romainseb) [inline .github/workflows/autopublish.yml:5]: We don't have a master branch on this repository
Let's give a try on gitflow there and not publish a release on develop

```suggestion
    branches: [main]
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+**What is the problem this PR is trying to solve?**
```
Comment (romainseb) [inline .github/PULL_REQUEST_TEMPLATE.md:1]: We can go for this template at first.
It's maybe a time a bit after to rethink if we want to change some stuff 🙂

File: .github/workflows/autopublish.yml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        run: |
+          alreadyPublished="$(yarn info @talend/react-faceted-search versions | sed -n "/'${{ steps.current-version.outputs.current_version }}'/ p" | wc -l)"
+          echo "Is local version already published ? $alreadyPublished"
+          echo "::set-output name=already-published::$alreadyPublished"
+
+      - name: Install
+        if: ${{ steps.check-already-published.outputs.already-published == 0 }}
+        run: yarn install --frozen-lockfile
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_AUTH_TOKEN }}
```
Comment (romainseb) [inline .github/workflows/autopublish.yml:38]: I think we don't need the NPM token for that as we are not supposed to fetch private dependencies

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -0,0 +1,32 @@
+name: Faceted search CI
+
+on:
+  pull_request:
+    branches: [master, develop, main]
```
Comment (romainseb) [inline .github/workflows/ci.yml:5]:
```suggestion
    branches: [develop, main]
```

File: .github/workflows/ci.yml [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
+          registry-url: 'https://registry.npmjs.org/'
+          scope: '@talend'
+
+      - name: Install
+        run: yarn install --frozen-lockfile
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_AUTH_TOKEN }}
```
Comment (romainseb) [inline .github/workflows/ci.yml:26]: I don't think we need it there as all faceted search dependencies are supposed to be public
```suggestion
```

**Review by romainseb — APPROVED** [review]

File: .github/workflows/autopublish.yml (hunk 3 above)
Comment (gjou-tlnd) [reply .github/workflows/autopublish.yml:38]: locale package is private, we'll fix it in the next release

**Pull request merged**